packages/edit-post/src/plugins/copy-content-menu-item/index.js

```diff
@@ -2,17 +2,27 @@
  * WordPress dependencies
  */
 import { ClipboardButton } from '@wordpress/components';
-import { withSelect } from '@wordpress/data';
+import { withDispatch, withSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import { withState, compose } from '@wordpress/compose';
 
-function CopyContentMenuItem( { editedPostContent, hasCopied, setState } ) {
+function CopyContentMenuItem( { createNotice, editedPostContent, hasCopied, setState } ) {
 	return (
 		<ClipboardButton
 			text={ editedPostContent }
 			role="menuitem"
 			className="components-menu-item__button"
-			onCopy={ () => setState( { hasCopied: true } ) }
+			onCopy={ () => {
+				setState( { hasCopied: true } );
+				createNotice(
+					'info',
+					'Content copied.',
```

Comment:
Maybe "All content copied" is better?

Reply:
Thanks, thats better. I updated the code.